src/ASP/frontend/modules/gamedata/js/game_mods.js

```diff
@@ -40,12 +40,12 @@
             rules: {
                 shortName: {
                     required: true,
-                    maxlength: 32,
+                    maxlength: 24,
                     nameRegex: true
                 },
                 longName: {
                     required: true,
-                    maxlength: 64
+                    maxlength: 48
```

Comment:
so this exactly matches the db varchar length?

Reply:
Yep, schema.sql is set at 24 and 48 for these columns so this is the default.

Reply:
i think its a good fix for validation of the expected string length.